yb-voyager/cmd/exportData.go

```diff
@@ -127,29 +128,78 @@ func exportDataCommandFn(cmd *cobra.Command, args []string) {
 
 	success := exportData()
 	if success {
-		tableRowCount := getExportedRowCountSnapshot(exportDir)
-		callhome.GetPayload(exportDir, migrationUUID)
-		callhome.UpdateDataStats(exportDir, tableRowCount)
-		callhome.PackAndSendPayload(exportDir)
+		packAndSendExportDataPayload(COMPLETE)
 
 		setDataIsExported()
 		color.Green("Export of data complete")
 		log.Info("Export of data completed.")
 		startFallBackSetupIfRequired()
 	} else if ProcessShutdownRequested {
 		log.Info("Shutting down as SIGINT/SIGTERM received.")
+		packAndSendExportDataPayload(EXIT)
 	} else {
 		color.Red("Export of data failed! Check %s/logs for more details.", exportDir)
 		log.Error("Export of data failed.")
+		packAndSendExportDataPayload(ERROR)
 		atexit.Exit(1)
 	}
 }
 
+func packAndSendExportDataPayload(status string) {
+	//TODO: send this INPROGRESS status in some fixed interval for long running export data
+
+	if !callhome.SendDiagnostics {
+		return
+	}
+	payload := createCallhomePayload()
+
+	switch exportType {
+	case SNAPSHOT_ONLY:
+		payload.MigrationType = OFFLINE
+	case SNAPSHOT_AND_CHANGES:
+		payload.MigrationType = LIVE_MIGRATION
```

Comment:
How/where are we telling in case FF/FB is enabled?

Reply:
It will come up with live migration-specific upcoming PR as anyways I can't get the information directly here about whether it will be with FF/FB it has to be extra which is TODO